phoenicis-dist/src/deb/control/control

```diff
@@ -1,5 +1,5 @@
 Package: phoenicis-playonlinux
-Version: 5.0
+Version: 5.0-SNAPSHOT
```

Comment:
We should remove the deb folder. Not a good idea to have two separate ways to build a deb.